contracts/sg-eth-airdrop/src/integration_tests.rs

```diff
@@ -0,0 +1,253 @@
+use cosmwasm_std::{Addr, Attribute};
+use cw_multi_test::error::Error;
+use cw_multi_test::{Contract, ContractWrapper, Executor, AppResponse};
+use ethers_core::k256::ecdsa::SigningKey;
+use ethers_core::types::H160;
+use sg_multi_test::StargazeApp;
+use sg_std::StargazeMsgWrapper;
+use std::str;
+
+use crate::msg::{AirdropEligibleResponse, ExecuteMsg, InstantiateMsg, QueryMsg};
+use crate::state::Config;
+use ethers_core::rand::thread_rng;
+use ethers_signers::{LocalWallet, Signer, Wallet, WalletError};
+use eyre::Result;
+
+const OWNER: &str = "admin0001";
+const AIRDROP_CONTRACT: &str = "contract0";
+
+fn custom_mock_app() -> StargazeApp {
+    StargazeApp::default()
+}
+
+pub fn contract() -> Box<dyn Contract<StargazeMsgWrapper>> {
+    let contract = ContractWrapper::new(
+        crate::contract::execute,
+        crate::contract::instantiate,
+        crate::contract::query,
+    );
+    Box::new(contract)
+}
+
+fn get_instantiate_contract<'a>(claim_plaintext: &str) -> StargazeApp {
+    let mut app = custom_mock_app();
+    let true_admin = Addr::unchecked("true_admin");
```

Comment:
probably better to define "true_admin" as a constant

Reply:
Now using OWNER for admin calls